instant-distance-py/Cargo.toml

```diff
@@ -11,12 +11,22 @@ repository = "https://github.com/InstantDomain/instant-distance"
 readme = "../README.md"
 
 [lib]
-name = "instant_distance"
-crate-type = ["cdylib"]
+name = "instant_distance_py"
+crate-type = ["cdylib", "lib"]
+
+[package.metadata.maturin]
+crate-type = ["cdylib", "lib"]
 
 [dependencies]
 bincode = "1.3.1"
-instant-distance = { version = "0.6", path = "../instant-distance", features = ["with-serde"] }
+instant-distance = { version = "0.6", path = "../instant-distance", features = [
```

Comment:
Please avoid unrelated formatting changes in `Cargo.toml`.

Reply:
vscode needs reigned in, as in imports as well.